### PR TITLE
10.16.0.183

### PR DESCRIPTION
**Pull request by BeateRixen**

### CODE REVIEW

File: content/release-10-16-0/applications-10-16-0-bundle/10-16-0-200.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+<tr>
+<td>Web SDK</td>
+<td>The <code>DatapointSelectorModule</code> can now be used in a module federation plugin.</td>
+<td>MTM-52251</td>
+</tr>
+
+<tr>
+<td>Web SDK</td>
+<td>An issue with the "copy dashboard" functionality has been addressed. The device context of a configured widget is now again replaced by the context it has been pasted to. This applies, for example, to the selected asset or to datapoints.</td>
```
Comment (rssag) [inline content/release-10-16-0/applications-10-16-0-bundle/10-16-0-200.md:50]: not too important, but i thought about swapping the ", for example," and the preposition ("to")

File: content/release-10-16-0/devices-10-16-0-bundle/10-16-0-200.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+<th>
+Description</th>
+<th>
+Issue</th>
+</tr>
+</thead><tbody>
+
+<tr>
+<td>Connectivity</td>
+<td>Due to a compatibility error of the connectivity microservice with the Kite provider, an error occurred on the metrics page, when the monthly consumption data was an out-of-scope integer. This has been fixed by changing the data type.</td>
```
Comment (rssag) [inline content/release-10-16-0/devices-10-16-0-bundle/10-16-0-200.md:25]: is it really two separate errors, or is the compatibility error showing up in the metrics page? then we should revise the entry and talk about a single error being raised

File: content/release-10-16-0/devices-10-16-0-bundle/10-16-0-200.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+<tr>
+<td>Device Management</td>
+<td>On bulk device registration views, the <b>Upload</b> button now stays disabled if there is no file selected or if the selected file is invalid.</td>
+<td>DM-1641</td>
+</tr>
+
+<tr>
+<td>Device Management</td>
+<td>On the <b>Services</b> tab in the device details, the filter for the "Status" column did not work properly and didn´t return any results when filtering was applied. This is now fixed.</td>
```
Comment (rssag) [inline content/release-10-16-0/devices-10-16-0-bundle/10-16-0-200.md:43]:
```suggestion
<td>On the <b>Services</b> tab in the device details, the filter for the "Status" column did not work properly and didn't return results when a filter was applied. This is now fixed.</td>
```

File: content/release-10-16-0/devices-10-16-0-bundle/10-16-0-200.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+<tr>
+<td>LWM2M</td>
+<td>Due to a limitation of the Leshan library used by the LWM2M agent, only device registration messages were supported where the LWM2M version is attached to the LWM2M object. The LWM2M agent has now been extended to also accept LWM2M registration messages where the LWM2M version is attached to the LWM2M object instance.</td>
+<td>DM-1893</td>
+</tr>
+
+<tr>
+<td>Loriot</td>
+<td>Events created by the Loriot agent when sending uplink requests will now be considered as device requests, providing connectivity status and availability monitoring.</td>
```
Comment (rssag) [inline content/release-10-16-0/devices-10-16-0-bundle/10-16-0-200.md:61]:
```suggestion
<td>Events created by the Loriot agent when sending uplink requests are now considered device requests, providing connectivity status and availability monitoring.</td>
```

File: content/release-10-16-0/platform-10-16-0-bundle/10-16-0-200.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+<tr>
+<td>Administration</td>
+<td>UTF-8 characters are now supported in names of files downloaded from the files repository with the export functionality.</td>
+<td>MTM-46346</td>
+</tr>
+
+<tr>
+<td>Authentication</td>
+<td>Fixed an issue with device requests counter being increased while switching between standard apps (Administration, Cockpit, Device management).</td>
```
Comment (rssag) [inline content/release-10-16-0/platform-10-16-0-bundle/10-16-0-200.md:32]:
```suggestion
<td>Fixed an issue with the device request counter being increased while switching between standard apps (Administration, Cockpit, Device management).</td>

```

File: content/release-10-16-0/applications-10-16-0-bundle/10-16-0-200.md (hunk 1 above)
Comment (BeateRixen) [reply content/release-10-16-0/applications-10-16-0-bundle/10-16-0-200.md:50]: That sounds weird then for me, will leave it :-).

File: content/release-10-16-0/devices-10-16-0-bundle/10-16-0-200.md (hunk 2 above)
Comment (BeateRixen) [reply content/release-10-16-0/devices-10-16-0-bundle/10-16-0-200.md:25]: Don´t get that. It is about one error. There is a compatibility error. And this leads to an error in the metric page.

Comment (BeateRixen) [inline content/release-10-16-0/devices-10-16-0-bundle/10-16-0-200.md:25]:
```suggestion
<td>Due to compatibility issues of the connectivity microservice with the Kite provider, an error occurred on the metrics page, when the monthly consumption data was an out-of-scope integer. This has been fixed by changing the data type.</td>
```

Comment (BeateRixen) [reply content/release-10-16-0/devices-10-16-0-bundle/10-16-0-200.md]: How about that?

**Review by rssag — APPROVED** [review]

File: content/release-10-16-0/devices-10-16-0-bundle/10-16-0-200.md (hunk 2 above)
Comment (rssag) [reply content/release-10-16-0/devices-10-16-0-bundle/10-16-0-200.md]: purrfect

**Pull request merged**